spyder/plugins/editor/widgets/base.py

```diff
@@ -31,11 +31,26 @@
 from spyder.py3compat import PY3, to_text_string
 from spyder.utils import icon_manager as ima
 from spyder.widgets.calltip import CallTipWidget, ToolTipWidget
+from spyder.widgets.helperwidgets import HTMLDelegate
 from spyder.widgets.mixins import BaseEditMixin
 from spyder.plugins.editor.api.decoration import TextDecoration, DRAW_ORDERS
 from spyder.plugins.editor.utils.decoration import TextDecorationsManager
 from spyder.plugins.completion.languageserver import CompletionItemKind
 
+COMPLETION_ITEM_TEMPLATE = """
+<table width="{width}" height="{height}">
+    <tr>
+        <td valign="middle" style="color:{color}">
+            {completion}
+        </td>
+
+        <td valign="middle" align="right" float="right" style="color:{color}">
+            {type}
+        </td>
+    </tr>
+</table>
+"""
+
 
 class CompletionWidget(QListWidget):

```

Comment:
@dalthviz, please move the completion widget and its utility functions out of here and to its own module (here in `editor/widgets`) called `completion.py`.